cw/capture.py

```diff
@@ -991,6 +1008,235 @@ def kmac_fvsr_key_batch(ctx: typer.Context,
     capture_end(ctx.obj.cfg)
 
 
+def capture_otbn_vertical(ot, ktp, fw_bin, pll_frequency, capture_cfg):
```

Comment:
I think we should either make the function name specific or move algorithm specific parts (generating input and comparing result) to specific functions.

Reply:
This is a good point. I totally agree.
It was name specific at first, but then we discussed in one of the sca meetings that it would be good to eventually support multiple algorithms. While moving algorithmic specific parts to specific functions, I came to the conclusion that this should be done at a later point when we actually have multiple algorithms.

So what's the best way for now?
- Change the name (here and in the config files) back, merge the PR, and then change it again, when we actually have multiple algorithms?
- Leave it as it is, knowing that there will be multiple algorithms soon? 
- Move the algorithmic specific parts to specific functions? (actually, I don't want to do this in this PR, as the PR is already too large)

@vogelpi what do you think?

Reply:
Let's stick to the generic name and move things around as we add support for other snippets. I also remember having discussed this before as @m-temp pointed out.

Reply:
I would leave it as is for now and do the name change in a fallow-up PR.
We are likely to have multiple capture functions related to otbn vertical attacks, so maybe we should discuss and agree on a naming strategy.